samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +59,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.None();
```

Comment:
Blame @HaoK for using over-simplistic names. I had used `SkipAuthentication()`, but something like `IgnoreAuthentication()` or `BypassAuthentication()` would be fine too.